frame/support/src/traits/dispatch.rs

```diff
@@ -101,6 +101,23 @@ pub trait UnfilteredDispatchable {
 	fn dispatch_bypass_filter(self, origin: Self::Origin) -> DispatchResultWithPostInfo;
 }
 
+/// Type that can be dispatched with an additional storage layer which is used to execute the call.
+pub trait DispatchableWithStorageLayer {
+	/// The origin type of the runtime, (i.e. `frame_system::Config::Origin`).
+	type Origin;
+
+	/// Same as `dispatch` from the `Dispatchable` trait, but specifically spawns a new storage
```

Comment:
```suggestion
	/// Same as `dispatch` from the [`Dispatchable`] trait, but specifically spawns a new storage
```

Reply:
Dispatchable is not in this crate. You want me to reference it from across crates?